android/java/org/chromium/chrome/browser/app/domain/KeyringModel.java

```diff
@@ -260,10 +256,10 @@ public void resetService(
         }
     }
 
-    public void getKeyringPerId(String keyringId, Callbacks.Callback1<KeyringInfo> callback) {
-        if (TextUtils.isEmpty(keyringId)) return;
+    public void getKeyringPerId(
+            @KeyringId.EnumType int keyringId, Callbacks.Callback1<KeyringInfo> callback) {
         KeyringInfo selectedCoinKeyring = _mSelectedCoinKeyringInfoLiveData.getValue();
-        if (selectedCoinKeyring != null && selectedCoinKeyring.id.equals(keyringId)) {
+        if (selectedCoinKeyring != null && selectedCoinKeyring.id == keyringId) {
```

Comment:
`selectedCoinKeyring.id == keyringId` : `==` will compare the references, use `equals` for String type in Java.

Reply:
But this is int now